src/vivarium_testing_utils/automated_validation/data_transformation/measures.py

```diff
@@ -175,7 +177,7 @@ def __init__(self, cause: str) -> None:
         )
 
     @utils.check_io(data=SingleNumericColumn, out=SingleNumericColumn)
-    def get_measure_data_from_artifact_gbd(self, data: pd.DataFrame) -> pd.DataFrame:
+    def get_measure_data_from_simulation_inputs(self, data: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Nit: can you be consistent here? Choose one of sim or simulation to use in these names